modules/custom-tuning-specification.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id='custom-tuning-specification-{context}']
+= Custom tuning specification
+
+[IMPORTANT]
+====
+Custom profiles are currently a Technology Preview feature.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+See the link:https://access.redhat.com/support/offerings/techpreview/[Red Hat
+Technology Preview features support scope] for more information.
+endif::[]
+====
+
+The custom resource (CR) for the operator has two major sections. The first
+section, `profile:`, is a list of tuned profiles and their names. The second,
+`recommend:`, defines the profile selection logic.
```

Comment:
The _custom resource_ the object, I think. Thus, should this be _The custom resource definition for the operator..._ ??

Reply:
I guess I'm fine with both, but the point was to have (CR) included here so we can refer to it later.